packages/fortinet_fortigate/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.12.0"
+  changes:
+    - description: Simplify script to split syslog message into a KV processor
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/
```

```suggestion
      link: https://github.com/elastic/integrations/pull/6398
```

CI is failing right now due to the missing PR number: `issue number in changelog link should be a positive number`